app/templates/components/modals/cfs-proposal-modal.hbs

```diff
@@ -8,13 +8,25 @@
     <p>{{t 'No existing sessions'}}</p>
   {{/if}}
   <div>
-    {{#link-to 'public.cfs.new-speaker' class='ui teal button' invokeAction=(action 'toggleView')}}
-      {{#if isNewSpeaker}}
+    {{#if isNewSpeaker}}
+      {{#link-to 'public.cfs.new-speaker' class='ui teal button' invokeAction=(action 'toggleView')}}
         {{t 'Add Speaker Details'}}
-      {{else}}
-        {{t 'Edit Speaker Details'}}
+      {{/link-to}}
+    {{else}}
+      {{#each data.userSpeaker as |speaker|}}
+        {{#if (eq speaker.email data.user.email)}}
+          {{speakerExists}}
```

Comment:
I think we should move this if condition logic to JS and use only one variable `speakerExists` for better readability.

Reply:
the computed property you tried to use should be something like this:
```js
speakerExists: computed('speaker.email', 'data.user.email', function() {
//do comparisons here and return the result to the computed property
})
```